src/utils/firebase.ts

```diff
@@ -1,7 +1,16 @@
 import { initializeApp } from "firebase/app";
 import { getFirestore } from "firebase/firestore";
 
-const firebaseConfig = {
+type Config = {
```

Comment:
- interface
- lowercase
- check to see if firebase alr has a config type u can use